compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -80,9 +83,11 @@ luci::CircleTranspose *create_4d_transpose(luci::CircleNode *node,
   for (uint32_t i = 0; i < 4; i++)
     perm->at<loco::DataType::S32>(i) = indices[i];
   perm->shape_status(luci::ShapeStatus::VALID);
+  perm->name(name + "/Transpose/perm");
```

Comment:
Two transpose nodes are created from one node. pre-transpose/post-transpose.

Both transpose may have the same name in this implementation. How about adding `indices` at the end?

For example, `perm->name(name + "/Transpose/perm_" + to_string(indices));`.

This will generate different names for pre/post Transposes.